package.json

```diff
@@ -19,11 +19,13 @@
     "test": "npm run test:keep && npm run test:strip",
     "test:all": "ember try:each",
     "test:keep": "ember test",
+    "pretest:node": "multidep node-tests/multidep.json",
     "test:node": "mocha node-tests",
     "test:strip": "STRIP_TEST_SELECTORS=true ember test"
   },
   "dependencies": {
-    "ember-cli-babel": "^5.1.7"
+    "ember-cli-babel": "^5.1.7",
```

Comment:
Should probably bump to ^6.0.0-beta.9 so that the addons own files are compiled with babel 6. That can definitely be done separately though...